AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -2996,7 +2996,7 @@ giveawayoftheday.com##.subscribe_block
 korrespondent.net##.subscribe_button
 aif.ru##.subscribe_content_wid
 beautyhack.ru##.subscribe_fix
-penzavzglyad.ru##.subscribe_form
+culture.ru,penzavzglyad.ru##.subscribe_form
```

Comment:
That is not enough. `culture.ru##.subscribe`, we have a common rule `##.subscribe`